.travis.yml

```diff
@@ -1,11 +1,13 @@
 language: cpp
 
-sudo: false
+sudo: required
+dist: precise
```

Comment:
Do you really need sudo and precise? It will make the time until the tests run much longer as travis as more `sudo: false` containers.